features/txt-parfmt-props.feature

```diff
@@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
   I need a ParagraphFormat object with read/write formatting properties
 
 
+  @wip
+  Scenario Outline: Get tab stops object
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops is a CT_TabStops object
+      
+    Examples:  values for paragraph_format.tab_stop elements 
+      | value |
+      | no    |
+      | some  |
+      
+
   Scenario Outline: Get paragraph alignment
     Given a paragraph format having <align-type> alignment
      Then paragraph_format.alignment is <value>
```

Comment:
Okay David, that should do it for feedback on this one. Let me know when you're ready and I'll try to be quicker the next time :)